ricecooker/sample_program.py

```diff
@@ -163,7 +163,7 @@ def construct_channel(**kwargs):
 
     channel = Channel(
         source_domain="learningequality.org",
-        channel_id="testing-ricecooker-channel",
+        source_id="testing-ricecooker-channel",
```

Comment:
Ah, yeah, this changes how people use it, too. We'll need to communicate the param name changes to the folks doing chefs.